packages/lib/src/vis/rgb/utils.ts

```diff
@@ -26,7 +26,7 @@ export function toRgbSafeNdArray(
 
   if (ndArr.dtype === 'int8') {
     return ndarray(
-      Uint8Array.from([...ndArr.data].map((val) => val + 128)),
+      Uint8Array.from(ndArr.data.map((val) => val + 128)),
```

Comment:
Unrelated: no need to copy the array since the `map` doesn't mutate the source array.

Reply:
:fishing_pole_and_fish: